packages/block-editor/src/components/block-list/use-block-props/use-event-handlers.js

```diff
@@ -142,5 +142,6 @@ export function useEventHandlers( ref, clientId ) {
 		insertDefaultBlock,
 		removeBlock,
 		selectBlock,
+		ref.current,
```

Comment:
This is not good, the issue with this is that the "ref.current" doesn't trigger a rerender when it changes. This means adding it here is almost useless (it can work if something else triggers a rerender at the same time). A better fix is to  refactor this hook to return a "ref" (see useFocusReturn for instance)